src/sagas/index.js

```diff
@@ -0,0 +1,55 @@
+import { all, call, put, takeLatest } from 'redux-saga/effects';
+
+function* sendReserveContent(action){
+    const reserveAction = action.payload.action;
+    const submitter = action.payload.submitter;
+    const node = action.payload.node;
+    const url = action.payload.url;
+
+    let json = action.payload.excelContent;
+    json = JSON.stringify(json);
+
+    let endpoint = 'http://localhost:8085/PDS_APIs/pds_doi_api/0.1/dois';
```

Comment:
This would need to be in a configuration file somewhere

Reply:
I'll move this out to a config.